superset-frontend/cypress-base/cypress/utils/index.ts

```diff
@@ -18,6 +18,8 @@
  */
 import { getChartAlias, Slice } from 'cypress/utils/vizPlugins';
 
+/* eslint-disable cypress/unsafe-to-chain-command */
```

Comment:
are these inline ignores something that we should be coming back around later to fix, or should we ignore them globally?

Reply:
idk the merit of that particular rule and it gave me quite a bit of trouble since I couldn't disable it globally for reasons beyond my understanding. 

Philosophically though, I think we need to move away from warnings as they muddy the output and slip over time. I think we should go full on boolean on lint rules.